sdk/daml-script/test/src/com/digitalasset/daml/lf/engine/script/test/MultiUpgradesIT.scala

```diff
@@ -13,16 +13,14 @@ import org.scalatest.Inside
 import org.scalatest.matchers.should.Matchers
 import org.scalatest.wordspec.AsyncWordSpec
 
-class UpgradesITDev extends AsyncWordSpec with AbstractScriptTest with Inside with Matchers {
+class MultiUpgradesIT extends AsyncWordSpec with AbstractScriptTest with Inside with Matchers {
```

Comment:
Note for future: we'll update this to match `main`, rename to `RuntimeUpgradesIT`, and perhaps even remove the existing `UpgradesIT`